packages/appengine/src/templates/timeZones.ts

```diff
@@ -1,560 +1,569 @@
 
 //https://gist.githubusercontent.com/ykessler/3349960/raw/05b3fc013d231189441f9c097368eb53dfe04876/timezones
+export function getTimeZonesFlatten() {
+    const flat = []
+    for(const groups of getTimeZones()) {
+        for(const zone of groups.zones) {
+            flat.push(zone)
+        }
+    }
+    return flat
+}
```

Comment:
Generators are really awesome and it's a good idea to get to know them. A Generator returns an iterator and you simply write a function that yields one value at a time. You can also just return and the iterator is undefined safe. So this could be something like:

```suggestion
export function *getTimeZonesFlatten() {
    for(const groups of getTimeZones())
        for(const zone of groups.zones)
            yield zone
}
```

and is used:

```
for(const zone of getTimeZonesFlatten())
```